doc/source/reference.rst

```diff
@@ -3,246 +3,105 @@
-
-  Returns a ``Series`` containing the area of each geometry in the ``GeoSeries``.
-
-.. attribute:: GeoSeries.bounds
-
-  Returns a ``DataFrame`` with columns ``minx``, ``miny``, ``maxx``,
-  ``maxy`` values containing the bounds for each geometry.
-  (see ``GeoSeries.total_bounds`` for the limits of the entire series).
-
-.. attribute:: GeoSeries.length
-
-  Returns a ``Series`` containing the length of each geometry.
-
-.. attribute:: GeoSeries.geom_type
+.. autoattribute:: geopandas.GeoSeries.area
 
-  Returns a ``Series`` of strings specifying the `Geometry Type` of
-  each object.
+.. autoattribute:: geopandas.GeoSeries.bounds
 
-.. method:: GeoSeries.distance(other)
+.. autoattribute:: geopandas.GeoSeries.length
 
-  Returns a ``Series`` containing the minimum distance to the `other`
-  ``GeoSeries`` (elementwise) or geometric object.
+.. autoattribute:: geopandas.GeoSeries.geom_type
 
-.. method:: GeoSeries.representative_point()
+.. automethod:: geopandas.GeoSeries.distance
 
-  Returns a ``GeoSeries`` of (cheaply computed) points that are
-  guaranteed to be within each geometry.
+.. automethod:: geopandas.GeoSeries.representative_point
 
-.. attribute:: GeoSeries.exterior
+.. autoattribute:: geopandas.GeoSeries.exterior
 
-  Returns a ``GeoSeries`` of LinearRings representing the outer
-  boundary of each polygon in the GeoSeries.  (Applies to GeoSeries
-  containing only Polygons).
-
-.. attribute:: GeoSeries.interiors
-
-  Returns a ``GeoSeries`` of InteriorRingSequences representing the
-  inner rings of each polygon in the GeoSeries.  (Applies to GeoSeries
-  containing only Polygons).
+.. autoattribute:: geopandas.GeoSeries.interiors
 
 `Unary Predicates`
 
-.. attribute:: GeoSeries.is_empty
-
-  Returns a ``Series`` of ``dtype('bool')`` with value ``True`` for
-  empty geometries.
-
-.. attribute:: GeoSeries.is_ring
-
-  Returns a ``Series`` of ``dtype('bool')`` with value ``True`` for
-  features that are closed.
-
-.. attribute:: GeoSeries.is_simple
+.. autoattribute:: geopandas.GeoSeries.is_empty
 
-  Returns a ``Series`` of ``dtype('bool')`` with value ``True`` for
-  geometries that do not cross themselves (meaningful only for
-  `LineStrings` and `LinearRings`).
+.. autoattribute:: geopandas.GeoSeries.is_ring
 
-.. attribute:: GeoSeries.is_valid
+.. autoattribute:: geopandas.GeoSeries.is_simple
 
-  Returns a ``Series`` of ``dtype('bool')`` with value ``True`` for
-  geometries that are valid.
+.. autoattribute:: geopandas.GeoSeries.is_valid
 
 `Binary Predicates`
 
-.. method:: GeoSeries.almost_equals(other[, decimal=6])
+.. automethod:: geopandas.GeoSeries.geom_almost_equals
 
-  Returns a ``Series`` of ``dtype('bool')`` with value ``True`` if
-  each object is approximately equal to the `other` at all
-  points to specified `decimal` place precision.  (See also :meth:`equals`)
+.. automethod:: geopandas.GeoSeries.contains
 
-.. method:: GeoSeries.contains(other)
+.. automethod:: geopandas.GeoSeries.crosses(other)
```

Comment:
Is there a reason you kept lines like `geopandas.GeoSeries.crosses(other)` when the format here matches the function definition? I noticed you removed some such instances (e.g., for `buffer`), but not all, and it wasn't clear to me why.

Reply:
I did at some point have an idea about removing all these, except for the functions that only pass on the `*args` and `**kwargs` (essentially `simplify` and `plot`). But I see that I did not consistently follow up on that.

I'll remove these, and add parameters for the `GeoSeries.plot`-function.